app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -319,6 +319,26 @@ def public_key=(key)
       attrs[:public_key] = key
     end
 
+    def normalize_vm_attrs(vm_attrs)
+      vm_attrs['cluster_id'] = vm_attrs.delete('cluster')
+      vm_attrs['cluster_name'] = self.clusters.detect { |c| c.id == vm_attrs['cluster_id'] }.try(:name)
+
+      vm_attrs['template_id'] = vm_attrs.delete('template')
```

Comment:
this would create template_name key with nil value if no template is found, in ec2 adapter we don't configure id attributes if strings are empty, should this be consistent everywhere?